### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by erickpintor**

### CODE REVIEW

File: FaunaDB.podspec [hunk 1]
```diff
@@ -1,16 +1,18 @@
 Pod::Spec.new do |s|
   s.name             = "FaunaDB"
-  s.version          = "1.0.0"
-  s.summary          = "Swift client for FaunaDB."
+  s.version          = "0.2.0-SNAPSHOT"
+>>>>>>> f029968... increasing version to 0.0.2-SNAPSHOT
```
Comment (marrony) [inline FaunaDB.podspec]: merge conflict?

Comment (erickpintor) [inline FaunaDB.podspec]: Owww... Hold on.

Comment (erickpintor) [inline FaunaDB.podspec]: Fixed.

**Comment by eaceaser** [comment]
lgtm

**Pull request merged**